internal/scorecard/alpha/tests/test_bundle.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/operator-framework/api/pkg/manifests"
+	"github.com/operator-framework/api/pkg/validation/errors"
+	"github.com/operator-framework/operator-registry/pkg/registry"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// TestBundle holds the bundle contents to be tested
+type TestBundle struct {
+	BundleErrors []errors.ManifestResult
+	Bundles      []*registry.Bundle
+	CRs          []unstructured.Unstructured
+}
```

Comment:
Can we make this struct look like we expect it to look once the API we want for parsing a bundle directory exists?

My guess is that something like the following will exist:
```go
func LoadBundleFromDirectory(bundlePath string) (*registry.Bundle, error) { ... }
```

With that in mind, can we just pass a `registry.Bundle` to the tests?

For getting CRs, we could either wrap `registry.Bundle` in a new type that has a method for getting the CRs, or we can just have a helper function that returns the CRs given a `registry.Bundle`.

Reply:
This is related to the other convo [here](https://github.com/operator-framework/operator-sdk/pull/2829#discussion_r407838390).

Again, totally fine to add a TODO to get back to this later so we're not blocked.

Just want to make sure the final thing we ship uses the new bundle format and has a straightforward/intuitive API.

Reply:
I added a TODO to address the API changes, at that point I like the idea of adding a helper to get the CRs if that is not part of the new API we eventually use.